.lintstagedrc.mjs

```diff
@@ -0,0 +1,4 @@
+export default {
+    '**/*.?(m)js': filenames => `eslint ${filenames.join(' ')} --fix`,
```

Comment:
I was debating whether I like the addition of `--fix` here 🤔 feels like it should be the developers responsibility if a commit passed through linting, as opposed to this where it automagically fixes it and the commit goes through if only autofixable errors / warnings are found. Hurts the discoverability a bit? But, not a blocker I guess 🤷